guidelines.md

```diff
@@ -54,23 +63,17 @@ In case of any error, the returned JSON object describes the error and has the f
 | `Message` | string | required | Description of the error. |
 | `Details` | string | optional | Additional details about the error \(request, headers, server stack trace, inner exceptions etc.\). Only available on development environment. |
 
-Some errors may also contain additional information relevant to the error on top of this two properties. But that depends on the operation and is specifically described in the operation documentation.
+Some errors may also contain additional information relevant to the error on top of these two properties. That depends on the operation and is specifically described in the operation documentation.
 
 ## Authentication
 
 Authentication and tokens that have to be used depending on environment.
 
 ### Demo environments
 
-These demo environments are meant to be used during implementation of the client applications. To sign into the system, use the following credentials:
-
-* **PlatformAddress** - `https://demo.mews.li`
```

Comment:
So how we will address the issue that partners can't log in to Commander to see their changes, thus being forced to develop the integration kind of blindly?

Reply:
We will provide them with the credentials privately when they start the process. It's not an issue to give it to an integration partner that is building to us. The issue was that it was available to everyone.

Reply:
Cool so I'd mention it here like: `You will receive credentials to Mews on demand.`

Reply:
@SrnaBegovic Did we change our approach to free access to demo? And when did it happen? We are presenting everywhere, that everyone can go to demo and start using our system.

Reply:
This is completely against what we wanted to have.

Reply:
@vitasamek Conversation [in all-announcements yesterday](https://mews.slack.com/archives/C02QXN5TL/p1589898889139100) about our competitors pulling client information from our system via demos. We thought this was a  good precaution to take. Any integration partner that wants to use our demo for the purpose of becoming a certified Mews partner has to go through us anyways. Partners will be given credentials on request.

Reply:
Yeah, maybe we could try figuring out some solution that both fixes the data leakage but also does not decrease user-friendliness (developer-friendliness) of the API usage.

Reply:
Actually, it can be provided to them in the instruction email once they complete the Mews new integration partner form. This way they have to at least register so we have them in Pipedrive as a partner in order to access the demo. I want to at least track who is using this in some way.

Reply:
It doesn't make sense to me. They have the accessToken and clientToken written in the documentation, they can pull whatever they want from via API - almost everything.

Reply:
Anyway, this is a public repo, if they want, they can find credentials in history. You should reset it.

Reply:
@vitasamek Reset what? Do you have any other suggestions on how to resolve this?

Reply:
If you what to hide the password, you need to also change the password.

Reply:
based on conversation with Vita and Honza in Slack, I'll revert the changes and request the review again.